src/shogun/clustering/KMeansBase.cpp

```diff
@@ -313,13 +313,17 @@ void CKMeansBase::init()
 	    &k, "k", "k, the number of clusters",
 	    ParameterProperties::HYPER | ParameterProperties::CONSTRAIN,
 	    SG_CONSTRAINT(positive<>()));
-	SG_ADD(&dimensions, "dimensions", "Dimensions of data");
-	SG_ADD(&fixed_centers, "fixed_centers", "Use fixed centers");
-	SG_ADD(&R, "radiuses", "Cluster radiuses");
+	SG_ADD(
+	    &dimensions, "dimensions", "Dimensions of data",
+	    ParameterProperties::MODEL);
```

Comment:
@karlnapf not sure if this is model?

Reply:
This is inferred from data or? Then read only

Reply:
yes, it is set with the number of features of the data. read only and model, or just read only?

Reply:
Just read only
Model parameters are those that are learnt by the train method